std/math.d

```diff
@@ -5035,9 +5035,14 @@ private:
             }
             else version (MIPS_Any)
             {
-                cast(void) __asm!uint("cfc1 $0, $$31\n" ~
-                                      "andi $0, $0, 0xFFFFFF80\n" ~
-                                      "ctc1 $0, $$31", "=r");
+                version (D_LP64)    enum mask = "0xFFFFFF80";
+                else                enum mask = "0xFF80";
+
+                cast(void) __asm!uint(`.set noat
```

Comment:
The original code used `"..."~"..."~...` which results in better asm output text alignment. I'd rather keep that with minor legibility cost here.

Reply:
There are other places in druntime using this style, I found it more pleasing to the eye then the old style.

I think you find yourself reading more of this D code than the ASM output of it, it makes it easier to understand for newbies also.

But I'm fine changing it back if this is the consensus.